static/src/javascripts/projects/common/modules/commercial/third-party-tags.js

```diff
@@ -14,8 +15,12 @@ define([
     'common/modules/commercial/third-party-tags/krux',
     'common/modules/identity/api',
     'common/modules/commercial/third-party-tags/outbrain',
-    'common/modules/commercial/third-party-tags/plista'
+    'common/modules/commercial/third-party-tags/plista',
+    'text!common/views/commercial/external-content.html',
+    'common/utils/fastdom-promise',
```

Comment:
Utils should be next to other utils just to keep it clean. If you look across another components you will find the pattern is:

Promise,
utils,
modules,
text